### ISSUE TITLE
Resizing columns in Catalog is very buggy

### ISSUE THREAD

**Issue by Xerus2000** [Xerus2000/monsterutilities#65]
oof this table is annoying ^^

**Comment by defvs** [comment]
Yeah I noticed it. But you get used to it, you just need to resize certain columns to resize others. It's weird. Can we fix it tho ?

**Comment by Xerus2000** [comment]
I dunno, that's why I created the issue instead of directly fixing it ^^ we might try other resize policies and default/max column sizes.
